catroid/src/main/java/org/catrobat/catroid/pocketmusic/ui/NoteView.java

```diff
@@ -48,10 +52,22 @@ public void onClick(View v) {
 	}
 
 	private int getToggledColor() {
-		return toggled ? blackColor : whiteColor;
+		if (toggled) {
+			return toggledColor;
+		} else {
+			return getUntoggledColor();
+		}
 	}
 
 	public boolean isToggled() {
 		return toggled;
 	}
+
+	public int getUntoggledColor() {
+		return untoggledColor;
+	}
+
+	public void setUntoggledColor(int untoggledColor) {
```

Comment:
Are you going to use this set method in the future? If that's the case - nevermind! If not: please remove it :)